lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
```

Comment:
Anyway, you can add this spec to `schopping_cart_spec``
```ruby
expect(subject).to respond_to(:has_items?)
```
and make it pass with the implementation I proposed ;-)